docs/topics/circuit-breaker-README.adoc

```diff
@@ -6,24 +6,89 @@ ifeval::["{runtime}" == "Spring Boot"]
 :OSORunCMD: mvn clean fabric8:deploy -Popenshift -DskipTests
 :guideURL: http://appdev.openshift.io/docs/spring-boot-runtime.html
 :fileLocation: src/main/resources/static/index.html
+:greeting-app-name: springboot-cb-greeting
+:name-app-name: springboot-cb-name
 endif::[]
 
 ifeval::["{runtime}" == "Eclipse Vert.x"]
 :localRunCMD: mvn vertx:run
 :OSORunCMD: mvn clean fabric8:deploy -Popenshift -DskipTests
 :guideURL: http://appdev.openshift.io/docs/vertx-runtime.html
 :fileLocation: src/main/resources/webroot/index.html
+:greeting-app-name: greeting-service
+:name-app-name: name-service
 endif::[]
 
 ifeval::["{runtime}" == "WildFly Swarm"]
 :localRunCMD: mvn wildfly-swarm:run
 :OSORunCMD: mvn clean fabric8:deploy -Popenshift -DskipTests
 :guideURL: http://appdev.openshift.io/docs/wf-swarm-runtime.html
 :fileLocation: src/main/webapp/index.html
+:greeting-app-name: greeting-service
+:name-app-name: name-service
 endif::[]
 
 
 
 = ${mission} - ${runtime} Booster
 
-IMPORTANT: This runs best when deployed to OpenShift in order to use the circuit breaker functionality. For more details on using this booster with a single-node OpenShift cluster, CI/CD deployments, as well as the rest of the runtime, see the link:{guideURL}[${runtime} Runtime Guide].
\ No newline at end of file
+IMPORTANT: This runs best when deployed to OpenShift in order to use the circuit breaker functionality. For more details on using this booster with a single-node OpenShift cluster, CI/CD deployments, as well as the rest of the runtime, see the link:{guideURL}[${runtime} Runtime Guide].
+
+== Run this Booster on a Single-node OpenShift Cluster
+If you have a single-node OpenShift cluster, such as Minishift or the Red Hat Container Development Kit, link:http://appdev.openshift.io/docs/minishift-installation.html[installed and running], you can also deploy your booster there. A single-node OpenShift cluster provides you with access to a cloud environment that is similar to a production environment.
+
+To deploy your booster to a running single-node OpenShift cluster:
+
+. Log in and create your project.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ git clone git@github.com:${loggedUser}/${targetRepository}
+
+$ cd ${targetRepository}
+
+$ oc login -u developer -p developer
+
+$ oc new-project MY_PROJECT_NAME
+
+$ {OSORunCMD}
+----
+
+
+== Interact with this Booster on a Single-node OpenShift Cluster
+
+To interact with your booster while it's running on a Single-node Openshift Cluster, you first need to obtain it's URL:
+
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get route {greeting-app-name} -o jsonpath={$.spec.host}
+
+{greeting-app-name}-MY_PROJECT_NAME.192.168.42.158.nip.io
+
+$ oc get route {name-app-name} -o jsonpath={$.spec.host}
+
+{name-app-name}-MY_PROJECT_NAME.192.168.42.158.nip.io
+----
+
+
+You can use the form from at browser at your application's url or you can use the `curl` command:
```

Comment:
I'm pretty sure "form from at browser at your application's url" is a mistake, but I don't know what exactly is wrong :-)